pom.xml

```diff
@@ -376,7 +376,7 @@
 			<dependency>
 				<groupId>javax.mail</groupId>
 				<artifactId>mail</artifactId>
-				<version>1.4.1</version>
+				<version>1.5.0-b01</version>
```

Comment:
I might suggest using version [1.6.2](https://search.maven.org/artifact/javax.mail/javax.mail-api/1.6.2/jar) with a slightly different artifact name.

Reply:
<groupId>javax.servlet.jsp.jstl</groupId>
				<artifactId>javax.servlet.jsp.jstl-api</artifactId>
				<version>1.2.2</version>
Big thanks @ibacher ,i change even the groupid . cc : @mozzy11 and @sherrif10